frontend/src/pages/projects/screens/detail/overview/ProjectOverview.tsx

```diff
@@ -21,7 +27,11 @@ const ProjectOverview: React.FC = () => {
         <PipelineAndVersionContextProvider>
           <TrainModelsSection />
         </PipelineAndVersionContextProvider>
-        {modelServingEnabled ? <ServeModelsSection /> : undefined}
+        {modelServingEnabled && serveModelsCard ? (
+          <LazyCodeRefComponent component={serveModelsCard} />
+        ) : (
+          <ServeModelsSection />
+        )}
```

Comment:
This is also duplicate logic inside of `frontend/src/concepts/projects/overviewTab/serveModels.tsx` so we only need it in one place. we can probably remove this one
```suggestion
        {modelServingEnabled &&  (
          <ServeModelsSection />
        )}
```

Reply:
When I make this change, my 'Serve Models' text is no longer displayed.

Reply:
Oh right gotcha, you'd have to use the `frontend/src/concepts/projects/overviewTab/serveModels.tsx` component `<ServeModelsCard />. Because right now it's not being used even though it's setup to handle showing the new and old versions 
```suggestion
        {modelServingEnabled && (
          <ServeModelsCard />
        )}
```

Reply:
Yeah, but doing this change caused this 'Start by deploying model' to appear 
```
{modelServingEnabled && (
          <ServeModelsCard />
        )}
```
I can do this import instead and it works as it's supposed to, but it looks funky
`import ServeModelsCard from '../../../../../../packages/modelServing/src/ServeModelsCard';`